scripts/skills/perks/perk_rf_fresh_and_furious.nut

```diff
@@ -12,6 +13,7 @@ this.perk_rf_fresh_and_furious <- ::inherit("scripts/skills/skill", {
 		this.m.Description = "This character is exceptionally fast when not fatigued.";
 		this.m.Icon = "ui/perks/rf_fresh_and_furious.png";
 		this.m.IconMini = "rf_fresh_and_furious_mini";
+		this.m.IconMiniBackup = this.m.IconMini;
```

Comment:
I would sort this line to the bottom of the block, even starting a new block with an empty line in between. That is to emphasize, that this/these variables are non-vanilla ones.